recipes/maxbin2/build.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
 set -euo pipefail
 cd src && make
+cp src/MaxBin $PREFIX/bin/
```

Comment:
Your script is already located in `src` at this point so `src/MaxBin` does not exist but `./MaxBin` does